src/main/java/core/basesyntax/figure/core/RightTriangle.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.figure.core;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure {
+    private double hypotenuse;
+    private double legA;
+    private double legB;
+
+    public RightTriangle(String colorOfFigure, double legA, double legB) {
+        super(colorOfFigure);
+        this.hypotenuse = Math.sqrt((legA * legA) + (legB * legB));
+        this.legA = legA;
+        this.legB = legB;
+    }
+
+    public RightTriangle(String colorOfFigure, Random random) {
+        this(colorOfFigure, random.nextDouble() * 10, random.nextDouble() * 10);
+    }
```

Comment:
let's figure this out with one constructor in each figure

Reply:
May I add Randome inside each constructor?